lib/ts/recipe/thirdparty/providers/custom.ts

```diff
@@ -312,7 +313,7 @@ export default function NewProvider(input: ProviderInput): TypeProvider {
                     await impl.config.validateIdTokenPayload({
                         idTokenPayload: rawUserInfoFromProvider.fromIdTokenPayload,
                         clientConfig: impl.config,
-                        userContext,
+                        userContext: userContext as UserContext,
```

Comment:
If the userContext passed in is not defined, we should call this function to make a usercontext: https://github.com/supertokens/supertokens-node/blob/master/lib/ts/utils.ts#L319